include/aktualizr-lite/cli/cli.h

```diff
@@ -34,12 +34,30 @@ enum class StatusCode {
   InstallRollbackFailed = 130,
 };
 
+/**
+ * The installation mode to be applied. Specified during InstallContext context initialization.
+ */
+enum class PullMode {
+  /**
+   * Default mode, do pull target during install operation.
+   */
+  PerformPull = 0,
```

Comment:
IMO, `PullMode::All` and `PullMode::None` looks a bit better, the `Pull` word in the enum value is redundant since the enum type denotes it.

Reply:
I was not happy with the current names. `All` and `None` are certainly better.

Reply:
Updated enum entries names.